duckduckgo/iOS
# fire pixels when the widgets are used

Task/Issue URL: https://app.asana.com/0/0/1205142282682119/f
Tech Design URL:
CC:

**Description**:

Fires pixels when the Lock Screen widgets are used.

**Steps to test this PR**:
1. Add each of the Lock Screen widgets, use them and check the pixels are fired.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
